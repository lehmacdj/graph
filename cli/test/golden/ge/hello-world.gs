==========input==========
mk hello
cd hello
mk world/"!"
cd world/"!"
cd @000000000000
ls
==========stdout==========
==========stderr==========
==========graph-dump==========
