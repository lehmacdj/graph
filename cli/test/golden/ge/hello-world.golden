==========stdout==========
g> g> g> g> g> g> "hello" at O8OhMSVtjevt
g> Goodbye!
==========stderr==========
==========graph-dump==========
000000000000{in=[0daCJjMrQel8 via "origin"], out=[O8OhMSVtjevt via "hello"]}
0daCJjMrQel8{in=[], out=[zsBuPkn5mh8F via "file-extensions",AhQufiPzgyRf via "file-hashes",3JJvxFUHGAA1 via "historical-special-nodes",S00KkOYoVpFu via "import-dates",a0fVkm0kR7KE via "import-urls",927u9Xyky2pC via "mime-types",000000000000 via "origin",pbYxBO6fzBQV via "tags"]}
3JJvxFUHGAA1{in=[0daCJjMrQel8 via "historical-special-nodes"], out=[VEfLhuTgZ88Z via "sequence-ids"]}
927u9Xyky2pC{in=[0daCJjMrQel8 via "mime-types"], out=[]}
AhQufiPzgyRf{in=[0daCJjMrQel8 via "file-hashes"], out=[]}
DQ1KCJgauHiV{in=[Pqf7kGOdKJIr via "01"], out=[sYumkceKpoAh via "01"]}
O8OhMSVtjevt{in=[sYumkceKpoAh via "00:00:00.000000000000",000000000000 via "hello"], out=[w4VxcgSm0Js6 via "world"]}
Pqf7kGOdKJIr{in=[S00KkOYoVpFu via "1970"], out=[DQ1KCJgauHiV via "01"]}
S00KkOYoVpFu{in=[0daCJjMrQel8 via "import-dates"], out=[Pqf7kGOdKJIr via "1970"]}
VEfLhuTgZ88Z{in=[3JJvxFUHGAA1 via "sequence-ids"], out=[]}
a0fVkm0kR7KE{in=[0daCJjMrQel8 via "import-urls"], out=[]}
pbYxBO6fzBQV{in=[0daCJjMrQel8 via "tags"], out=[]}
sYumkceKpoAh{in=[DQ1KCJgauHiV via "01"], out=[O8OhMSVtjevt via "00:00:00.000000000000",w4P7BSd8jU2v via "00:00:01.000000000000",w4VxcgSm0Js6 via "00:00:01.000000000000"]}
w4P7BSd8jU2v{in=[w4VxcgSm0Js6 via "!",sYumkceKpoAh via "00:00:01.000000000000"], out=[]}
w4VxcgSm0Js6{in=[sYumkceKpoAh via "00:00:01.000000000000",O8OhMSVtjevt via "world"], out=[w4P7BSd8jU2v via "!"]}
zsBuPkn5mh8F{in=[0daCJjMrQel8 via "file-extensions"], out=[]}
